picard/ui/coverartbox.py

```diff
@@ -255,28 +283,29 @@ def load_remote_image(self, url, mime, data):
         try:
             coverartimage = CoverArtImage(
                 url=url.toString(),
+                types=[u'front'],
                 data=data
             )
         except CoverArtImageError as e:
             log.warning("Can't load image: %s" % unicode(e))
             return
         if isinstance(self.item, Album):
             album = self.item
-            album.metadata.append_image(coverartimage)
+            album.metadata.set_front_image(coverartimage)
```

Comment:
But, depending on chosen option, the other front images should be kept imho. What about either prepending dropped image, or replacing only the first one ? And keep the rest as is.